lnd_test.go

```diff
@@ -851,7 +862,7 @@ func testUpdateChannelPolicy(net *lntest.NetworkHarness, t *harnessTest) {
 	defer close(bQuit)
 
 	chanAmt := maxBtcFundingAmount
-	pushAmt := btcutil.Amount(100000)
+	pushAmt := chanAmt / 2
```

Comment:
Since the channel is opened in the opposite direction now (see comment below), the `pushAmt` must be increased to make sure the link `Bob->Carol` has capacity to carry the payments in this test. Since the channel reserve is based on the total channel capacity, not changing this would make the `reserve > pushAmt`.